### PR TITLE
Mega tweaks

### PR DESCRIPTION
**Pull request by stephenthoma**
Made a lot of small tweaks to the site styling and changed the text wording in a few sections.

### CODE REVIEW

**Comment by thejinxters** [comment]
Cool, I'll check it in a bit!

File: _includes/header.html [hunk 1]
```diff
@@ -4,9 +4,9 @@
             <h1>Join the Tandem Revolution</h1>
             <h3>Build meaningful connections in your community</h3>
             <hr>
-            <p>Having influential people in your life is important and Influencers in your community want to share their time with people like you. We connect you with a community “influencers” so that you can have a quality conversation about the things that matter to you most.</p>
-            {% include form.html %}
-            <a href="#why" class="btn btn-default btn-xl page-scroll">Or Find Out More</a>
+            <p>Having influential people in your life is important. Let usconnect you with our community of “influencers” so that you can have quality conversations about the things that matter the most to you.</p>
```
Comment (thejinxters) [inline _includes/header.html]: Let "US CONNECT"  typo

**Comment by thejinxters** [comment]
We should chat this afternoon. I think you made some good changes but there are a few things we might want to tweak before merging. Example: we need a real background image, do we want a sign up form at the top (usually you want to convert traffic as fast as possible)? Otherwise, looks good!

**Comment by Zandrr** [comment]
Gotta say the made with :heart: thing is some trendy bullshit that I don't like.  Other than that, :shipit:

**Pull request merged**